backend/src/modules/tournament-tables/core/application/TournamentTablesRepository.ts

```diff
@@ -0,0 +1,11 @@
+import { TournamentTable } from '../domain/TournamentTable';
+
+export interface TournamentTablesRepository {
+  save(tournamentTable: TournamentTable): Promise<void>;
+
+  saveAll(tournamentTables: TournamentTable[]): Promise<void>;
+
+  findByTournamentId(tournamentId: string): Promise<TournamentTable[] | undefined>;
+
+  findAll(): Promise<TournamentTable[]>;
```

Comment:
Repozytorium powinno mieć tylko metody, które są przydatne dla use casów aplikacji. Czy to gdzieś tam będzie przydatne :) ? No i nie jest to testowane, bo obecnie żaden CommandHandler / QueryHandler przez to nie przechodzi. Zdaje mi się, że nie ma sensu zwracanie wszystkich stołów, bo u nas stół ma sens w kontekście turnieju, czyli metoda wyszukiwania po id turnieju powinna wystarczyć.